example-singleServer-noBackend.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>HTML5 Speedtest</title>
+<meta name="viewport" content="width=device-width">
+<link rel="shortcut icon" href="favicon.ico">
+<script src="speedtest.js"></script>
+<script>
+function I(i){return document.getElementById(i);}
+
+//INITIALIZE SPEEDTEST
+var s=new Speedtest(); // create speedtest object
+
+//CUSTOM SETTINGS for version with no PHP or other server-side scripting
+
+// we only want ping, download and upload test
+s.setParameter("test_order", "P_D_U");
+// disable telemetry
+s.setParameter("telemetry_level", "0");
```

Comment:
0 is not a valid telemetry level (`none` is the correct value).
You can remove this line entirely because telemetry is disabled by default.

Reply:
fixed